installer/install.iss

```diff
@@ -114,6 +114,10 @@ Name: assoc_sh; Description: Associate .sh files to be run with Bash; Types: def
 Name: consolefont; Description: Use a TrueType font in all console windows; OnlyBelowVersion: 10.0
 Name: autoupdate; Description: Check daily for Git for Windows updates
 Name: windowsterminal; Description: "(NEW!) Add a Git Bash Profile to Windows Terminal"; MinVersion: 10.0.18362
+#ifdef WITH_EXPERIMENTAL_SCALAR
+Name: scalar; Description: "(NEW!) Scalar"; Types: default
```

Comment:
This seems a little short on words. Can we give the user a bit of a hint what scalar is? A hint that this is experimental (and not the same scalar they find by googling "scalar git" (.NET scalar)) might also be helpfull.

Reply:
That's a good point.

Unfortunately, I don't think there's an easy way to embed links e.g. to https://github.com/microsoft/scalar#readme.

And also: I am currently a bit busy with -rc2, and then have to fix bugs in this PR (I tested without building the full installer so far, and naturally that hid the bug where `scalar reconfigure --all` must not be run as administrator because it wouldn't find the enlistments).

After that, I will try to come up with a better description.

Please note, however, that the description is not likely to matter a whole lot, as the component will _only_ be shown if the installer does include Scalar's files. And if that is already the case, the idea is to turn this on by default. And the checkbox is there only out of an abundance of caution: I would like users not to turn it off unless they have a _really_ good reason for that (one valid reason would be a bug that forces them to go back to .NET-based Scalar).

Reply:
Maybe steal the very initial "Scalar is a project that tests out new strategies to make Git scale better. " Then something about it meaning very large repositories, and maybe selecting a commonly known one that's just large enough (Chromium, Android, whatever) to count (rather than just being "Micro$oft Mega Repo" ;-)

Reply:
For some ideas about how to briefly describe Scalar, perhaps consider [the Philosophy of Scalar](https://github.com/microsoft/scalar/blob/main/docs/philosophy.md), specifically this sentence:

> Scalar takes out the guesswork by automatically configuring your Git repositories to take advantage of the latest and greatest features.

Reply:
I don't really think that the Components page has enough space for such a description:

<img width=75% src=https://user-images.githubusercontent.com/127790/120209504-0ea6dc80-c22f-11eb-9da1-bb33e1722094.png />

As you can easily see, something like "Scalar is a project that tests out new strategies to make Git scale better" would most likely be way too long, and in any case inconsistent with the existing descriptions.

Reply:
How about this?

> Scalar (Git add-on to manage large-scale repositories)

Reply:
Sounds good.

Reply:
This is how it looks now:

<img width=75% src=https://user-images.githubusercontent.com/127790/120323522-db735480-c2e5-11eb-90d7-e77d7ccc4eb9.png />